packages/dds/merge-tree/src/localReference.ts

```diff
@@ -262,9 +271,13 @@ export class LocalReferenceCollection {
 	}
 
 	/**
+	 * Returns an iterator.
```

Comment:
Can we be more specific here? What is the returned iterator intended to iterate?